index.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+
+<body>
+
+    <header>
+        <nav class="navigation">
+            <p class="white-1 roboto-f l-height-24 f-size-18">John Doe</p>
+            <img class="menu" src="hamburguer.png" alt="">
+        </nav>
+    </header>
+
+
+    <div class="wrapper">
+
+        <div class="profile">
+            <p class="orange crete-f f-weight400 l-height-46 f-size-56">Hey There. I'm Gerson
+            <p>
```

Comment:
- [ ] for better SEO performance, the main title `Hey There. I'm Gerson` should be an `h1` tag instead of `<p>` tag , kindly fix it and you can check this [link ](https://www.reliablesoft.net/h1-tag/) for more infos